src/Provision.js

```diff
@@ -83,10 +83,15 @@ const InitializeTenant = async ({client, kmsId, tenantId, debug=false}) => {
     formatArguments: true,
   });
 
+  //tenant admin group and content admin group are both already created by ElvSpace/TenantCreate, just need to retreive them.
```

Comment:
I think we should retrieve the tenant_admins group from the contract just like the content admins below.

We currently read it from userProfileClient but we should not rely on that (because we want to deprecate that feature)